lm_eval/models/translators.py

```diff
@@ -44,18 +44,30 @@ def generate_until(self, reqs: list[Instance]) -> list[str]:
                 "target_lang": self.target_lang,
                 "model": self.model
             }
-
-            response = requests.post(self.endpoint,
-                                     headers=self.headers,
-                                     json=data)
-
-            if response.status_code != 200:
-                print(f"An error occurred while translating: {response.text}")
-                raise Exception(response.text)
-
-            result = response.json()
-            results.append(result['translations'][0]['text'])
-
+            max_attempts = 10
+            attempts = 0
+            while attempts < max_attempts:
+                try:
+                    response = requests.post(self.endpoint,
+                                             headers=self.headers,
+                                             json=data)
+
+                    if response.status_code != 200:
+                        print(f"An error occurred while generating: {response.text}")
+                        raise Exception(response.text)
+
+                    result = response.json()
+                    results.append(result['choices'][0]['message']["content"])
+                    attempts = 0
+                    break
+                except Exception as e:
+                    attempts += 1
+                    print("error occured. retrying...")
```

Comment:
It may be good to print the exception, so it's easy to debug what causes it.
For example, maybe we can ignore temporary network error, but if the error were caused by wrong credential, we can easily spot it.

Reply:
Thank you! I'll print exception!